sdk/keyvault/keyvault-admin/recordings/node/keyvaultbackupclient/recording_beginbackup.js

```diff
@@ -29,32 +29,32 @@ nock('https://eastus2.keyvault_name.managedhsm-int.azure-int.net:443', {"encoded
   'no-cache' ]);
 
 nock('https://eastus2.keyvault_name.managedhsm-int.azure-int.net:443', {"encodedQueryParams":true})
-  .post('/backup', {"storageResourceUri":"https://chrissprim.blob.core.windows.net/backup","token":"?sv=2019-12-12&ss=bfqt&srt=sco&sp=rwdlacupx&se=2022-07-14T05:56:52Z&st=2020-09-03T21:56:52Z&spr=https,http&sig=Rzts1hBG%2BgqTCTZo%2BX%2FZgJWJ5Ao16ueN%2F1BB4Dg8%2FLo%3D"})
+  .post('/backup', {"storageResourceUri":"https://chrissprim.blob.core.windows.net/backup","token":"blob_storage_sas_token"})
```

Comment:
Is the resource public? `chrissprim` ?

Reply:
It's not public. It's a test resource though. Let me ping @christothes for awareness.

Reply:
I just rotated the keys

Reply:
Better to mask it in the recordings.

Reply:
yes, I assumed we would also sanitize the recordings.

Reply:
To be clear, I thought we were talking about the sas token leak - are you suggesting we should mask the resource name also?

Reply:
We're not making this endpoint opaque in .Net, but I'll make it opaque in JS. Please don't consider this blocking.

Reply:
I rotated the keys when I spotted the sas token leak last week. This is just the resource name.

Reply:
I'm suggesting to mask the resource name too.

Reply:
Done!